generators/client/templates/angular/package.json.ejs

```diff
@@ -155,14 +140,8 @@ limitations under the License.
     <%_ if (protractorTests) { _%>
     "webdriver-manager": "12.1.7",
     <%_ } _%>
-    "webpack": "VERSION_MANAGED_BY_CLIENT_ANGULAR",
     "webpack-bundle-analyzer": "VERSION_MANAGED_BY_CLIENT_ANGULAR",
-    "webpack-cli": "VERSION_MANAGED_BY_CLIENT_ANGULAR",
-    "webpack-dev-server": "VERSION_MANAGED_BY_CLIENT_ANGULAR",
-    "webpack-merge": "VERSION_MANAGED_BY_CLIENT_ANGULAR",
-    "webpack-notifier": "VERSION_MANAGED_BY_CLIENT_ANGULAR",
```

Comment:
Other frameworks are using `webpack-notifier`, shouldn't we keep using this also in Angular part? If dropping this functionality then needs to be deleted and cleaned up also file `webpack/logo-jhipster.png`.

Reply:
I don't see why I removed it, let's put it back. Btw it seems that we don't use it in Vue...